level1c4pps/seviri2pps_lib.py

```diff
@@ -470,7 +488,7 @@ def get_encoding_seviri(scene):
     encoding['time'] = {'units': 'days since 2004-01-01 00:00',
                         'calendar': 'standard',
                         '_FillValue': None,
-                        'chunksizes': [1]}
+                        'chunksizes': (1)}
```

Comment:
```suggestion
                        'chunksizes': (1,)}
```

###### Explanation
This results in a very confusing error when using `-ne netcdf4`:
```Python-traceback
  File "xarray/backends/netCDF4_.py", line 497, in prepare_variable
  File "src/netCDF4/_netCDF4.pyx", line 2962, in netCDF4._netCDF4.Dataset.createVariable
  File "src/netCDF4/_netCDF4.pyx", line 4283, in netCDF4._netCDF4.Variable.__init__
TypeError: object of type 'int' has no len()
```

The reason is that `[1]` results in a list with one entry 1, but `(1)` does result in an int with value 1. Using the trailing comma tells python to construct a tuple with one element. (“tuples are not formed by the parentheses, but rather by use of the comma”, see the [python docs](https://docs.python.org/3.11/reference/expressions.html#parenthesized-forms))

Alternatively, if you don’t like the trailing comma:
```suggestion
                        'chunksizes': tuple([1])}
```

or (probably the worse option), continue using the list
```suggestion
                        'chunksizes': [1]}
```